scss/modules/_datatables.scss

```diff
@@ -253,3 +278,42 @@
     @include cell-width($width);
   }
 }
+
+@include media($med) {
+  .column--state {
+    width: 50px;
+  }
+
+  .column--small {
+    width: 90px;
+  }
+
+  .column--med {
+    width: 15%;
+  }
+
+  .column--large {
+    width: 20%;
+  }
+
+  .column--xl {
+    width: 30%;
+  }
+
+  .panel-active {
+    .column--xl,
+    .column--large {
+      width: 50%;
+    }
+
+    .column--med {
+      width: 25%;
+    }
+  }
+}
```

Comment:
Yeah I struggled with if it made sense to use those classes or these. I ultimately decided the more generic ones made sense because they'd be different sizes at different screen sizes. I'm not opposed to keeping the others in here if they're useful, but it might be good to settle on an approach. Do you have strong feelings one way or the other?